### ISSUE TITLE
Overclock is possible?

### ISSUE THREAD

**Issue by ivan89el** [mgba-emu/mgba#2833]
Some games slow down or drop out frames, mostly 3D/2.5D. Overclocking the emulated processor would solve this problem.  Sorry for my English.

**Comment by ChromiusMixin** [comment]
I don't know if you can overclock the CPU with speeding up audio since that's what happens on real hardware, but maybe tightening timings on the memory can do just that?

Isn't there an FPGA GBA core that gets better performance by doing that?

**Comment by endrift** [comment]
Yeah, you can't purely overclock the CPU without causing timing issues with other hardware. There are ways to "overclock" the RAM, but that only gets you a little bit more performance.

**Comment by QcSamuel** [comment]
the fpga gba core has a turbo mode(overclock) could be useful to have something similar on mgba for games that dont speed up and homebrew games

**Comment by endrift** [comment]
Which FPGA GBA core? There are multiple.

**Comment by QcSamuel** [comment]
just checked and mame has a cpu overclock option tried openlara and got solid 30fps with it

**Comment by ChromiusMixin** [comment]
MAME GBA also isn't very good for a fair amount of games, Mario & Luigi Superstar Saga crashes for me on MAME as soon as it attempts the first battle transition in the prologue.

**Comment by SuperDavid2** [comment]
With the overclock feature on MAME GBA, it is possible to reach 60fps on Doom 2, but the graphics are a mess. The first Doom runs too fast if it's overclocked, but Doom 2 appears to run at the correct speed.

**Comment by Hydra-Spectre** [comment]
I want this too.
